platforms/software/dependency-management/src/integTest/groovy/org/gradle/integtests/resolve/ResolutionIssuesIntegrationTest.groovy

```diff
@@ -447,54 +314,6 @@ class ResolutionIssuesIntegrationTest extends AbstractIntegrationSpec {
         succeeds("resolve")
     }
 
-    @NotYetImplemented
-    @Issue("https://github.com/gradle/gradle/issues/14220#issuecomment-2008178522")
-    def "another reproducer for corrupt RR"() {
```

Comment:
🤔 Looks like this test fully disappeared while it was fixed by the changes here. 
Should we keep it as the simpler regression test?
And name it according to the explanation of what happens.

Reply:
This test now lives in `VersionConflictResolutionIntegrationTest`